### DISCUSSION TITLE
How to clear all registers?

### DISCUSSION THREAD

**Discussion by otakutyrant** [neovim/neovim#24388] [Q&A] [answered]
I have copy something to some registers. Now I want to clear them to keep registers clear.

**Comment by fynnfluegge** [comment, marked as answer]
There is no default command to clear all the registers you filled previously.

A common approach is to use this kind of function:

```
command! WipeReg for i in range(34,122) | silent! call setreg(nr2char(i), []) | endfor
```

Call `:WipeReg` and this will iterate over all registers and sets them to empty.

**Reply by otakutyrant** [reply]
Thank you.
